### DISCUSSION TITLE
구독 버튼 및 댓글 작성에 네모 박스로 나옵니다.

### DISCUSSION THREAD

**Discussion by zzocojoa** [pronist/hELLO#70] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: mac
- 브라우저: chrom
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 3.6.1

**설명**
구독 버튼 및 댓글 작성이 네모 박스로 나옵니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

**스크린샷 및 링크**
아래 사진은 본문 내용 하단에 댓글 창입니다.
![image](https://user-images.githubusercontent.com/52736095/159676087-3b1e2a43-dbc0-4bd7-ae49-71a872fceb9e.png)

아래 사진은 오른쪽 상단의 구독버튼입니다.
![image](https://user-images.githubusercontent.com/52736095/159676418-b8dda962-b327-41db-ace9-873cb3489d42.png)

**Comment by pronist** [comment]
@zzocojoa 블로그 주소 남겨주세요~

**Comment by zzocojoa** [comment]
@pronist https://zzcojoa.tistory.com
입니다!!!

**Comment by pronist** [comment, marked as answer]
@zzocojoa 혹시 스킨 커스터마이징하셔서 폰트 임의로 변경하셨나요? 아이콘이 사용된 부분은 **폰트 어썸(Font Awesome)**이 사용되었기 때문에 전용 폰트가 아니라 다른 폰트가 적용되면 나타나지 않습니다. 따라서 이건 원본 코드에서 나타난 것이 아닌 임의 수정에 의한 버그로 판단됩니다.

**Comment by zzocojoa** [comment]
@pronist 답변감사합니다. 다른 사람이 올린 오류를 살펴봐도 없길래 어떤 문제인가 싶었는데 폰트변경때문이였군요. ㅎㅎ 배포해주신 스킨 감사히 잘 사용하겠습니다^^
